### DISCUSSION TITLE
l2p icon?

### DISCUSSION THREAD

**Discussion by rkendry** [tgorgdotcom/locast2plex#149] [General]
First of, thank you for putting this together. it's been a great addition to our cord cutting strategy.
Silly question, is there an icon for locast2plex?
When i load it in my unraid box it just has a question mark.

**Comment by tgorgdotcom** [comment]
Not a silly question at all.  Unfortunately I do not at the moment.  I do have some style knowledge, but not a lot of time.  If anyone wants to send some ideas along in this thread, maybe we can work out a design.
